src/styles/blocks/header.scss

```diff
@@ -0,0 +1,16 @@
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px #eee;
```

Comment:
Consider using a variable for colors (e.g., `#eee`) to improve maintainability and readability of the code. You can define color variables in a separate file like `variables.scss` and then import them.